File: Java Baekjoon/src/b1712/main_1712.java
package b1712;

import java.util.Scanner;

public class main_1712 {
    public static void main(String[] args) {
        int cost_A; // 1000
        int cost_B; // 70
        int cost_C; // 1070


        // User Input.
        Scanner sc = new Scanner(System.in);

        cost_A = sc.nextInt();
        cost_B = sc.nextInt();
        cost_C = sc.nextInt();


        // Break-Even Point Cal.
        if (cost_C <= cost_B) {
            System.out.println(-1);
        }
        else {
            int num = cost_A / (cost_C - cost_B);
            System.out.println(num+1);
        }


        sc.close();
    }
}

File: Python Baekjoon/b1712/main_1712.py
## 손익분기점
import sys

def main() :
    cost_A, cost_B, cost_C = map(int, sys.stdin.readline().split())

    if (cost_C <= cost_B) :
        print(-1)
    else :
        num = (cost_A // (cost_C - cost_B))
        print(num + 1)



if (__name__ == "__main__") :
    main()